hoodie-client/src/main/java/com/uber/hoodie/config/HoodieStorageConfig.java

```diff
@@ -44,6 +44,9 @@
   public static final String PARQUET_COMPRESSION_RATIO = "hoodie.parquet.compression.ratio";
   // Default compression ratio for parquet
   public static final String DEFAULT_STREAM_COMPRESSION_RATIO = String.valueOf(0.1);
+  public static final String LOGFILE_TO_PARQUET_COMPRESSION_RATIO = "hoodie.logfile.to.parquet.compression.ratio";
```

Comment:
The next PR is expected to have rolling stats related changes.

Reply:
Not really. I also explained this to @bvaradar but failed to update here. Essentially, this config is to determine the ratio between the on disk size of log files vs parquet files. This will continue to remain as a heuristic for now. The next PR will have rolling stats but that is to address the small file problem which is orthogonal to this PR at the moment.